libs/logger/logger.cpp

```diff
@@ -15,4 +15,30 @@ limitations under the License.
 */
 
 // This file is used to link dependencies library
-#include "logger.hpp"
\ No newline at end of file
+#include "logger.hpp"
+
+namespace logger {
+
+    // To hide spdlog
+    void setGlobalLogLevel(const level &l) {
```

Comment:
What reason to have four level for system? May be we handle only two global log levels: debug and release?